content/en/blog/posts/2022-09-09-build-release-v0.11.0.md

```diff
@@ -0,0 +1,84 @@
+---
+title: "Shipwright v0.11.0 Is Here"
+date: 2022-09-09T12:00:01-04:00
+draft: false
+author: "Sascha Schwarze ([@SaschaSchwarze0](https://github.com/SaschaSchwarze0))"
+---
+
+Summer is over, and Shipwright is back with the v0.11.0 release.
```

Comment:
Nit: Technically speaking, our friends on the other hemisphere could disagree with it being summer :)

Reply:
Good point.